process/factory/shard/vmContainerFactory_test.go

```diff
@@ -4,8 +4,7 @@ import (
 	"sync"
 	"testing"
 
-	ipcNodePart1p2 "github.com/ElrondNetwork/arwen-wasm-vm/v1_2/ipc/nodepart"
-	arwenConfig "github.com/ElrondNetwork/arwen-wasm-vm/v1_4/config"
+	arwenConfig "github.com/ElrondNetwork/wasm-vm/config"
 	"github.com/ElrondNetwork/elrond-go-core/core"
 	"github.com/ElrondNetwork/elrond-go-core/data"
```

Comment:
goimports

Reply:
Fixing on a separate branch, for all files in the repo.